model/textparse/openmetricsparse_test.go

```diff
@@ -684,3 +778,214 @@ func TestOMNullByteHandling(t *testing.T) {
 		require.Equal(t, c.err, err.Error(), "test %d", i)
 	}
 }
+
+func TestCTParseFailures(t *testing.T) {
+	input := `# HELP something Histogram with _created between buckets and summary
+# TYPE something histogram
```

Comment:
Also awesome to see tests asserting our known failure modes. Do we want to open issues and work on them in the following weeks/months?

If we're not working on it anytime soon, what do you think about adding a comment explaining the situation here? Something in the lines of "while not desirable, we are aware of a few limitations and these tests show them... TODO, fix stuff"

Reply:
Added this